src/optimise/FitResult.h

```diff
@@ -47,6 +48,9 @@ class FitResult{
     double GetExtremeVal() const;
     void   SetExtremeVal(double);
 
+    void SetCovarianceMatrix(const DenseMatrix);
+    const DenseMatrix GetCovarianceMatrix() const;
```

Comment:
Just FYI - the first `const` doesn't do much when you're returning by value. No need to change it though
https://stackoverflow.com/questions/8716330/purpose-of-returning-by-const-value